BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@io_bazel_rules_go//go:def.bzl", "gazelle", "go_library", "go_prefix")
+
+go_prefix("github.com/atlassian/smith")
+
+gazelle(
```

Comment:
gazelle is a tool that keeps lists of source files in `BUILD.bazel` files up to date. See https://github.com/bazelbuild/rules_go/#generating-build-files